packages/coil-client/src/graphQlClient.ts

```diff
@@ -38,7 +38,7 @@ export class GraphQlClient {
 
   public constructor(
     @inject(GraphQlClientOptions)
-    private config: GraphQlClientOptions = new GraphQlClientOptions()
+    protected config: GraphQlClientOptions = new GraphQlClientOptions()
```

Comment:
Is this change needed?

Reply:
no, not needed, but it's not "incorrect"
it was cause I'd modified these files previously in another branch
so I just pinched the changes
I'm not religious about confining small changes to their own commits